src/containers/App.jsx

```diff
@@ -34,14 +57,17 @@ function mapStateToProps(state) {
 
 function mapDispatchToProps(dispatch) {
   return {
-    locationChanged: bindActionCreators({locationChanged}, dispatch)
+    locationChanged: bindActionCreators(locationChanged, dispatch)
```

Comment:
and this remains as written and sort of becomes a manifest of event handlers we're threading through props as the number of events we're concerned with scales to N